xrpl4j-crypto-parent/xrpl4j-crypto-core/src/main/java/org/xrpl/xrpl4j/crypto/core/keys/Seed.java

```diff
@@ -64,6 +64,10 @@ public static Seed secp256k1SeedFromPassphrase(final Passphrase passphrase) {
     return secp256k1SeedFromEntropy(Entropy.of(entropyBytes));
   }
 
+  public static Seed ed25519Seed() {
```

Comment:
Plz add javadoc to describe that entropy is created here using something like `{@link Entropy#foo}`